packages/slate-react/src/components/content.js

```diff
@@ -206,7 +207,7 @@ class Content extends React.Component {
     native.removeAllRanges()
 
     // COMPAT: IE 11 does not support Selection.extend
```

Comment:
Comment should be updated.

Reply:
Done!

Reply:
Great, but also keep the IE-comment about not supporting range.extend please.